src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -6674,6 +6676,18 @@ namespace HeatBalanceManager {
             state.dataSurface->FrameDivider(FrameDividerNum).DividerSolAbsorp = FrameDividerProps(16);
             state.dataSurface->FrameDivider(FrameDividerNum).DividerVisAbsorp = FrameDividerProps(17);
             state.dataSurface->FrameDivider(FrameDividerNum).DividerEmis = FrameDividerProps(18);
+            state.dataSurface->FrameDivider(FrameDividerNum).NfrcProductType = DataSurfaces::NfrcProductOptions::CurtainWall;
+
+            // look up the NFRC Product Type for Assembly Calculations using the DataSurfaces::NfrcProductName
+            // state.dataSurface->FrameDivider(FrameDividerNum).NfrcProductType =
+            //     static_cast<DataSurfaces::NfrcProductOptions>(getEnumerationValue(static_cast<gsl::span<std::string_view>>(DataSurfaces::NfrcProductName),
```

Comment:
Is there a reason `getEnumerationValue` didn't work here?

Reply:
I never could get the syntax right.